Frontend/src/Components/Temperature/TempView.js

```diff
@@ -0,0 +1,70 @@
+import {Flex, VStack, Box} from "@chakra-ui/react";
+import RangeRow from "../Shared/RangeRow";
+import CONSTANTS from "../../data-constants.json";
+
+
+export default function TempView(props) {
```

Comment:
```suggestion
export default function Temperature(props) {
```
nit: Let's change the component name to `Temperature` or `Temperatures` rather than `TempView`